trexit/lib/trexit/endpoint.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Trexit.Endpoint do
+  use Plug.Router
+
+  plug :match
+  plug :dispatch
+
+  get "/schedules/:route_id" do
+    json(conn, Trexit.GTFS.schedules_for_route(route_id))
+  end
+
+  defp json(conn, json) do
+    conn
+    |> put_resp_content_type("application/json")
+    |> send_resp(200, Jsonrs.encode_to_iodata!(json))
```

Comment:
Note that for Jsonrs, there is no difference between `encode!` and `encode_to_iodata!`.

Also, we could add the option `lean: true` which would disable usage of the custom struct encoding protocol (which we are not using) and be more efficient ([docs](https://hexdocs.pm/jsonrs/readme.html#quick-start)).